lib/msal-common/src/client/BaseClient.ts

```diff
@@ -78,6 +85,7 @@ export abstract class BaseClient {
         headers.set(`${AADServerParamKeys.X_CLIENT_VER}`, this.config.libraryInfo.version);
         headers.set(`${AADServerParamKeys.X_CLIENT_OS}`, this.config.libraryInfo.os);
         headers.set(`${AADServerParamKeys.X_CLIENT_CPU}`, this.config.libraryInfo.cpu);
+        headers.set(HeaderNames.X_MS_LIB_CAPABILITY, HeaderNames.X_MS_LIB_CAPABILITY_VALUE);
```

Comment:
Why is this added here and in `createDefaultTokenRequestHeaders`?

Reply:
This is a new header the server wants to receive for each request to basically tell the server that we support throttling. It's outlined in the original spec.

Reply:
Sure but the question is, why is it being added twice? Once as part of the default library headers and again as part of the default token request headers?

Reply:
Question still stands. This headers is added on Line 72 and on 88

Reply:
Sorry didn't see the reply here--I assumed this was two different places where headers get created, or does the token request headers just get added to the default library headers?

Reply:
On second though we probably only need them to be on the token requests anyway, as that's the only time we're looking to throttle a new thumbprint. Will change

Reply:
Changed, it now takes clientId as a parameter and uses that instead